go.mod

```
@@ -22,21 +21,133 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/prometheus/client_golang v1.11.0
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
-	golang.org/x/sys v0.0.0-20210817190340-bfb29a6856f2
+	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5
+	golang.org/x/sys v0.0.0-20211029165221-6e7872819dc8
 	google.golang.org/grpc v1.42.0
-	k8s.io/api v0.22.4
-	k8s.io/apimachinery v0.22.4
+	google.golang.org/protobuf v1.27.1
+	k8s.io/api v0.23.0
+	k8s.io/apimachinery v0.23.0
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/cloud-provider v0.22.4
-	k8s.io/klog/v2 v2.10.0
+	k8s.io/cloud-provider v0.23.0
+	k8s.io/klog/v2 v2.30.0
 	//
 	// when updating k8s.io/kubernetes, make sure to update the replace section too
 	//
-	k8s.io/kubernetes v1.22.4
-	k8s.io/mount-utils v0.22.4
-	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a
-	sigs.k8s.io/controller-runtime v0.10.3
+	k8s.io/kubernetes v1.23.0
+	k8s.io/mount-utils v0.23.0
+	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
+	sigs.k8s.io/controller-runtime v0.11.0-beta.0.0.20211208212546-f236f0345ad2
```

```suggestion
	sigs.k8s.io/controller-runtime v0.11.0
```
let's wait till we have a controller-runtime stable release ?

---

we don't know when we will have the next stable release. and more over as we are sticking to a commit i think we are good we are already sticking to a commit in multiple places.

---

I think this is fine. When controller-runtime releases a new version, dependabot will send a PR to update it.